src/commands/apply.js

```diff
@@ -25,7 +25,7 @@ module.exports = class extends BaseCommand {
     async execute(client, message, args) {
         await message.delete();
         const type = args[0].toLowerCase();
-        if (message.channel.parentID === botsettings.category) return message.channel.send(messages.wrongChannel);
+        if (message.channel.parentID !=== botsettings.category) return message.channel.send(messages.wrongChannel);

```

Comment:
you just added `!` btw, so this statement became an invalid syntax lol

Reply:
Yuh 😅 minus 1 =